public/api-documentation/swagger.json

```diff
@@ -262,6 +262,46 @@
         }
       }
     },
+    "/manifests/{id}/cromerr_submit_transaction": {
```

Comment:
Let's keep CROMERR out of the API. We don't mention it anywhere else. It isn't a business function specific to that one endpoint. Isn't the entire app in service to CROMERR?